backend/internal/controllers/book.go

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import (
+	"encoding/json"
+	"github.com/go-chi/chi"
+	"net/http"
+
+	"github.com/TritonSE/words-alive/internal/database"
+	//"github.com/TritonSE/words-alive/internal/models"
+)
+
+type BookController struct {
+	Books database.BookDatabase
+}
+
+// Fetches a list of all books sorted by title for the main page (/books)
+func (c *BookController) GetBookList(rw http.ResponseWriter, req *http.Request) {
+	books, err := c.Books.FetchBookList(req.Context())
+	if err != nil {
+		rw.WriteHeader(500)
```

Comment:
I think it might be a good idea to use the `http` package status code constants, just since not all of us have them memorized yet :) https://golang.org/pkg/net/http/#pkg-constants

Can you replace 500 with http.StatusInternalServerError?